src/MeetingsSDKAdapter.js

```diff
@@ -0,0 +1,153 @@
+import {MeetingsAdapter} from '@webex/component-adapter-interfaces';
+import {Observable, fromEvent, concat} from 'rxjs';
+import {filter, finalize, map, publishReplay, refCount} from 'rxjs/operators';
+
+const LOCAL = 'local';
```

Comment:
Could we use a more descriptive constant name?

```suggestion
const MEDIA_TYPE_LOCAL = 'local';
```